### ISSUE TITLE
Create SIP Domain missing parameters

### ISSUE THREAD

**Issue by mcelicalderon** [twilio/twilio-php#320]
*Note: These issues are for bugs and feature requests for the helper libraries. If you need help or support, please email help@twilio.com and one of our experts will assist you!*

**Version:** 5.0.0-RC6
**API Subdomain (api/taskrouter/ip_messaging):** API

### Code Snippet
```php
$ipAccessControlListMapping = $client
    ->account
    ->sip
    ->domains
    ->create(
        "dunder-mifflin-scranton.sip.twilio.com",
        array(
            "friendlyName" => "Scranton Office",
            "voiceUrl" => "https://dundermifflin.example.com/twilio/app.php",
            "authType" => "IP_ACL"
        )
    );
```

Currently there is no interface in v5 of this library to create a new SIP Domain with all the allowed parameters. Specifically `authType`

https://github.com/twilio/twilio-php/blob/next-gen/Twilio/Rest/Api/V2010/Account/Sip/DomainList.php#L119

**Comment by ihumanable** [comment]
Great catch, this is missing in our definition file, we will update it.

**Comment by jingming** [comment]
Released with `5.0.0-RC9`.
